test/unittests/api/v0.0.1/candidates/Visit_Test.php

```diff
@@ -89,6 +89,7 @@ function testPutInvalidBattery() {
         $this->assertEquals($API->Headers, ["HTTP/1.1 400 Bad Request"]);
     }
     function testPostValidVisit() {
+        \User::singleton("admin");
```

Comment:
However, when the code is "out there", it needs to do said validation, which `APIBase` does. Returns `401` if the user is not logged in.